packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts

```diff
@@ -258,19 +261,51 @@ class IssuePrepaidCardWorkflow extends Workflow {
     }),
   ]);
 
-  constructor(owner: unknown) {
+  constructor(owner: unknown, workflowPersistenceId: string) {
     super(owner);
+    this.workflowPersistenceId = workflowPersistenceId;
+
     this.attachWorkflow();
   }
+
+  restoreFromPersistedWorkflow() {
+    this.session.restoreFromStorage();
+
+    const lastCompletedCardName = this.session.state.lastCompletedCardName;
+
+    // TODO: Only attempt to restore when both layer 1 and 2 are connected
+    if (lastCompletedCardName) {
+      const postables = this.milestones
+        .flatMap((m: any) => m.postableCollection.postables)
+        .concat(this.epilogue.postables);
+
+      const index = postables.mapBy('cardName').indexOf(lastCompletedCardName);
+
+      postables.slice(0, index + 1).forEach((p: any) => {
+        p.isComplete = true;
+      });
+    }
+  }
```

Comment:
This doesn't cover the case of canceled workflows yet.

Reply:
Forgot about these. Will add this